src/transactionManagers/SafeAppsSdkTransactionManager/index.ts

```diff
@@ -15,11 +14,10 @@ class SafeAppsSdkTransactionManager implements TransactionManager {
 
   async execTransactions({
     appsSdk,
-    transactions
-  }: ExecTransactionSafeAppsProps): Promise<TransactionResult> {
-    appsSdk.sendTransactions(transactions)
-
-    return new Promise((resolve, reject) => resolve({ hash: 'unknown' }))
+    transactions,
+    requestId
+  }: ExecTransactionSafeAppsProps): Promise<void> {
```

Comment:
It doesn't seem like this TransactionManager is meant to be used directly, right? If so, then exposing the `transactionManager` on the CPK instances actually causes `execTransactions` to be inferred to return `TransactionResult | void` instead of simply `TransactionResult`, even though in practice the `void` option will never be exercised because the `SafeAppsSdkConnector` will preemptively return a `TransactionResult` when the CPK detects that it's being used in a Safe Apps context.